Fix occ twofactor (#4209)
owncloud/docs: modules/admin_manual/pages/configuration/user/user_auth_twofactor.adoc

= User Two-Factor Authentication
:toc: right

== Introduction

> With two-factor authentication (2FA), the user can access his ownCloud web account only by using a trusted device like his mobile phone. When the user wants to sign in, he needs to provide two pieces of information (factors):

Reviewer: ```suggestion
With two-factor authentication (2FA), users can access their ownCloud web accounts only by using a trusted device like their mobile phone. When users want to sign in, they need to provide two pieces of information (factors):
```

> * His password

Reviewer: ```suggestion
* the password,
```

> * The six-digit verification code that's automatically displayed on his trusted devices or sent to his phone number.

Reviewer: ```suggestion
* the six-digit verification code that's automatically displayed on the trusted device or sent to the phone number.
```

== Setting Up 2FA

To provide 2FA functionality, an app like the {oc-marketplace-url}/apps/twofactor_totp[2-Factor Authentication] needs to be installed and enabled.

If a two-factor provider app is enabled, it is enabled for all users by default *but* _a user has to opt-in_, though the provider can decide whether or not the user has to pass the challenge.

== Troubleshooting

> === Tasks by the User

Reviewer: ```suggestion
=== Tasks for the User
```

Because the user has to opt-in, see the xref:user_manual:personal_settings/security.adoc[Security section in Personal Settings] link for tasks on the user side.

=== Second Factor is Inaccessible

> In case a user is losing access to the second factor like a lost or defect phone with two-factor SMS/app verification, the user would now be locked out. To give the user access to his account, an admin can temporarily disable the two-factor check _for that user_ via the

Reviewer: ```suggestion
In case a user loses access to the second factor, e.g. by breaking or losing the phone with two-factor SMS/app verification, the user is locked out. To give the user access to the account again, an admin can temporarily disable the two-factor check _for that user_ via the
```

xref:configuration/server/occ_command.adoc#two-factor-authentication[occ commands for Two-Factor Authentication]. 
After the issue has been fixed, the admin can reenable two-factor authentication for that user.

=== Manage Secrets

> When owncloud's {oc-marketplace-url}/apps/twofactor_totp[2-Factor Authentication] is used, the admin can manage

Reviewer: ```suggestion
If owncloud's {oc-marketplace-url}/apps/twofactor_totp[2-Factor Authentication] is used, the admin can manage
```
